src/theme.js

```diff
@@ -171,6 +171,7 @@ export default {
   backgroundColors,
   borderColors,
   borderWidth,
+  boxShadows,
```

Comment:
To match styled-system's theme, boxShadows is exported lower as shadows

Reply:
:+1: